common/spec/dependabot/pull_request_creator/message_builder_spec.rb

```diff
@@ -2649,6 +2649,196 @@ def commits_details(base:, head:)
+            stub_request(:get, %r{https://api\.github\.com/repos/gocardless/.+/contents/})
+              .to_return(status: 200, body: fixture("github", "business_files.json"), headers: json_header)
+            stub_request(:get, %r{https://api\.github\.com/repos/gocardless/.+/releases\?per_page=100})
+              .to_return(status: 200, body: fixture("github", "business_releases.json"), headers: json_header)
+            stub_request(:get, %r{https://api\.github\.com/repos/gocardless/.+/contents/CHANGELOG\.md\?ref=master})
+              .to_return(status: 200, body: fixture("github", "changelog_contents.json"), headers: json_header)
+            stub_request(:get, %r{https://rubygems\.org/api/v1/gems/.+\.json})
+              .to_return(status: 200, body: fixture("ruby", "rubygems_response_statesman.json"))
+            stub_request(:get, %r{https://github\.com/gocardless/.+\.git/info/refs\?service=git-upload-pack})
+              .to_return(
+                status: 200,
+                body: fixture("git", "upload_packs", "no_tags"),
+                headers: {
+                  "content-type" => "application/x-git-upload-pack-advertisement"
+                }
+              )
+          end
+
+          it "has the correct message" do
+            expected_message = "Bumps the go_modules group with 6 updates in the /foo directory:\n\n" \
+                               "| Package | From | To |\n" \
+                               "| --- | --- | --- |\n" \
+                               "| [business2](https://github.com/gocardless/business2) | `2.4.0` | `2.5.0` |\n" \
+                               "| [business3](https://github.com/gocardless/business3) | `3.4.0` | `3.5.0` |\n" \
+                               "| [business4](https://github.com/gocardless/business4) | `4.4.0` | `4.5.0` |\n" \
+                               "| [business5](https://github.com/gocardless/business5) | `5.4.0` | `5.5.0` |\n" \
+                               "| [business6](https://github.com/gocardless/business6) | `6.4.0` | `6.5.0` |\n" \
+                               "| [business](https://github.com/gocardless/business) | `1.4.0` | `1.5.0` |\n\n" \
+                               "Bumps the go_modules group with 5 updates in the /bar directory:\n\n" \
+                               "| Package | From | To |\n" \
+                               "| --- | --- | --- |\n" \
+                               "| [business2](https://github.com/gocardless/business2) | `2.4.0` | `2.5.0` |\n" \
+                               "| [business3](https://github.com/gocardless/business3) | `3.4.0` | `3.5.0` |\n" \
+                               "| [business4](https://github.com/gocardless/business4) | `4.4.0` | `4.5.0` |\n" \
+                               "| [business5](https://github.com/gocardless/business5) | `5.4.0` | `5.5.0` |\n" \
+                               "| [business6](https://github.com/gocardless/business6) | `6.4.0` | `6.5.0`"
```

Comment:
This also looks good! It renders out to:

---

Bumps the go_modules group with 6 updates in the /foo directory:

| Package | From | To |
| --- | --- | --- |
| [business2](https://github.com/gocardless/business2) | `2.4.0` | `2.5.0` |
| [business3](https://github.com/gocardless/business3) | `3.4.0` | `3.5.0` |
| [business4](https://github.com/gocardless/business4) | `4.4.0` | `4.5.0` |
| [business5](https://github.com/gocardless/business5) | `5.4.0` | `5.5.0` |
| [business6](https://github.com/gocardless/business6) | `6.4.0` | `6.5.0` |
| [business](https://github.com/gocardless/business) | `1.4.0` | `1.5.0` |

Bumps the go_modules group with 5 updates in the /bar directory:

| Package | From | To |
| --- | --- | --- |
| [business2](https://github.com/gocardless/business2) | `2.4.0` | `2.5.0` |
| [business3](https://github.com/gocardless/business3) | `3.4.0` | `3.5.0` |
| [business4](https://github.com/gocardless/business4) | `4.4.0` | `4.5.0` |
| [business5](https://github.com/gocardless/business5) | `5.4.0` | `5.5.0` |
| [business6](https://github.com/gocardless/business6) | `6.4.0` | `6.5.0`